docs/source/_developers/primer.md

```diff
@@ -0,0 +1,234 @@
+
+##  Writing Style
+The writing style in documentation is critical for ensuring consistency and understanding. Here are some best practices:
+
+**Clarity**: Be as clear and straightforward as possible. Avoid ambiguity. Documentation is not a place for complex sentences and unusual words.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - The initiation of the replication process...
+  - To start the replication process...
+:::
+
+**Conciseness**: Keep sentences and paragraphs short. This makes text easier to read and understand.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - Once the process has been completed successfully, the next step is...
+  - After completing the process, next...
+:::
+
+**Active Voice**: Use the active voice whenever possible as it is more straightforward and engaging.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - The command can be executed by the user.
+  - The user can execute the command.
+:::
+
+**Address the Reader**: Use the second person ("you") to address the reader directly.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - One can see the results by... <br>
+  - You can see the results by...<br>
+:::
+
+- Avoid using jargon and slang, as they can be confusing.
+- Keep your writing easy to read by using short sentences and paragraphs.
+- If you need to explain complex concepts, break them into smaller, more manageable sections to help your reader understand your message.
+
+##  Formatting
+MyST allows for markdown to be used for our Spinx project (i.e., RTD).
+
+:::{admonition} Bookmark and Review-worth Reference
+Check out this [Markdown cheatsheet].
+:::
+
+
+Commonly used elements follow the [MyST-Typography].
+
+Directives allow us to insert and format various elements (e.g., images, callouts, etc.). We use `colon_fence` to replace the typical hyphens Learn more about  [Colon Fence].
+
+:::::{important}
+We use the `colon_fence` extension the markdown rendered by Myst parses three colons (i.e., *:*) to open and close blocks. ([about colon_fence]).
+
+::::{note}
+[Myst-Parser Documentation] includes several constructs we can use to enhance our pages. **Use elements with purpose, whether to call attention, reference related content, or warn.
+::::
+
+Nesting blocks, along with the ability set parameters inline.
+
+::::{note}
+There is no need to indent source text, as nested directives are explicitly clear by the number of `:` characters are used to open and close.
+::::
+
+::::{attention}
+
+**Do not add line-breaks unless a line break in the rendered view is to need a line break.** Each paragraph remains on the same line, with line-breaks using to clearly show transitions between paragraphs, sections, and directives.
+
+:::{tip}
+Set *Soft-Wrap* to be set to 100 characters by default when viewing `*.md`
```

Comment:
We should also include how to set this up in VSCode (I found it in settings and searching wrap) since we have everyone setup an environment in VSCode in the initial setup of the python environment to build the HTML files locally.

Reply:
I like this. I will leave this for the README.

Well, do you think the README is a good spot for it, and then we can point to that in the primer? See README on this branch: https://github.com/northeastern-rc/rc-public-documentation/tree/joe/styleguide

Reply:
I think that would be a perfect spot. I found it helpful to set it after we discussed about it. Thank you!!